play-json/jvm/src/test/scala/play/api/libs/json/JsonSpec.scala

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright (C) 2009-2016 Lightbend Inc. <https://www.lightbend.com>
+ */
+package play.api.libs.json
+
+import java.util.{ Calendar, Date, TimeZone }
+
+import com.fasterxml.jackson.databind.{ JsonNode, ObjectMapper }
+import play.api.libs.functional.syntax._
+import play.api.libs.json.Json._
+
+import scala.collection.immutable.ListMap
+
+class JsonSpec extends org.specs2.mutable.Specification {
```

Comment:
Specs for `Json` which are specific to the JVM env, about Jackson types, `java.time` and `BigDecimal` used not to loose precision on numeric values (contrary to the JS env)